device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/fileupload/FileUpload.java

```diff
@@ -150,13 +149,13 @@ public synchronized void execute(IotHubStatusCode status, Object context)
                 catch (ClassCastException | NullPointerException | UnsupportedOperationException e)
                 {
                     /* Codes_SRS_FILEUPLOAD_21_023: [If the FileUploadStatusCallBack failed to delete the `FileUploadInProgress`, it shall log a error.] */
-                    logger.LogError("FileUploadStatusCallBack received callback for unknown FileUpload");
+                    log.error("FileUploadStatusCallBack received callback for unknown FileUpload");
                 }
             }
             else
             {
                 /* Codes_SRS_FILEUPLOAD_21_022: [If the received context is not type of `FileUploadInProgress`, the FileUploadStatusCallBack shall log a error and ignore the message.] */
-                logger.LogError("FileUploadStatusCallBack received callback for unknown FileUpload");
+                log.error("FileUploadStatusCallBack received callback for unknown FileUpload");
```

Comment:
As above

Reply:
There is no exception at this level, so I cannot.